src/snarky.d.ts

```diff
@@ -1274,7 +1279,7 @@ declare class Ledger {
   /**
    * Signs an account update.
    */
-  static signAccountUpdate(
+  static signOtherAccountUpdate(
```

Comment:
why did this name change?

Reply:
This is an unrelated fix, the name was actually `signOtherAccountUpdate` in the ocaml bindings, so it didn't work before